mycroft/skills/skill_manager.py

```diff
@@ -138,8 +193,9 @@ def run(self):
         self._load_on_startup()
 
         # Update sync backend and skills.
-        self.skill_updater.post_manifest(reload_skills_manifest=True)
-        self.settings_downloader.download()
+        if is_paired():
+            self.skill_updater.post_manifest(reload_skills_manifest=True)
```

Comment:
These two lines are duplicated in handle_paired().  Can we just call that method here?  Alternatively, write a new method both handle_paired and this method can call?